src/platform/DeviceControlServer.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *          Provides the implementation of the DeviceControlServer object.
+ */
+
+#include <platform/internal/DeviceControlServer.h>
+
+#include <platform/ConfigurationManager.h>
+
+namespace chip {
+namespace DeviceLayer {
+namespace Internal {
+
+DeviceControlServer DeviceControlServer::sInstance;
+
+CHIP_ERROR DeviceControlServer::ArmFailSafe(uint16_t expiryLengthSeconds)
+{
+    // TODO
+    return CHIP_NO_ERROR;
```

Comment:
Can we return a `CHIP_ERROR_NOT_IMPLEMENTED ` until something is implemented  ? (same comment for the other empty methods).

Reply:
Acked